source/about/cloud-subscriptions.rst

```diff
@@ -4,7 +4,9 @@ Cloud subscriptions
 Purchase a Cloud subscription
 ------------------------------
 
-Mattermost Cloud subscriptions start with a 14-day trial. When you're near the end of your trial, a banner within Mattermost will notify you to upgrade to a subscription plan.
+Mattermost Cloud can be used free forever with Mattermost Starter. 
+
+You can also start a free, 30-day, trial of Mattermost Enterprise to explore additional features. When you're near the end of your trial, a banner within Mattermost will notify you to upgrade to a subscription plan. You can upgrade to a monthly per-user subscription, with Mattermost Professional and Mattermost Enterprise.
 
 1. Select **Subscribe now** from the banner.
 2. Provide your credit card details. 
```

Comment:
These steps only work for upgrading to Professional. To upgrade to Enterprise, you must contact sales.